lib/src/http_request_impl.dart

```diff
@@ -6,8 +6,13 @@ import 'exception.dart';
 const bool _kIsWeb = bool.fromEnvironment('dart.library.js_util');
 
 class HttpRequestImpl implements HttpRequest {
-  HttpRequestImpl(this.serverUrl, this.apiKey, [this.connectTimeout])
-      : dio = Dio(
+  HttpRequestImpl(
```

Comment:
Just checking, if the user uses the new constructor, all the headers, auth, user agent, and others will still be sent correctly, right?

Reply:
yes they are going to be the same, since the customDio logic is using this constructor as well